avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDe.java

```diff
@@ -41,6 +43,8 @@
 
   private static final Map<String, Schema> primitiveSchemas;
   protected SchemaRegistryClient schemaRegistry;
+  protected SubjectNameStrategy keySubjectNameStrategy = new TopicNameStrategy();
```

Comment:
nit : default is already specified in the config def. So, this is not necessary.

Reply:
It is necessary: if the field is not initialised here, we get the NPEs that @rhauch [complained about](https://github.com/confluentinc/schema-registry/pull/680#issuecomment-355602913) in tests for the kafka-connect-avro-converter module. It might be that those tests aren't properly configuring the SerDe, but I didn't want to get into debugging tests that are unrelated to the feature at hand.